### DISCUSSION TITLE
Zsh completions missing in macOS app package contents

### DISCUSSION THREAD

**Discussion by phillip-hirsch** [ghostty-org/ghostty#2988] [Feature Requests, Ideas] [answered]
The zsh completions added in #2925 did not export them to the macOS package contents:
<img width="1032" alt="Screenshot 2024-12-16 at 7 28 16 PM" src="https://github.com/user-attachments/assets/e99dba5b-8237-4dbe-86b5-bf01a16b5735" />

Screenshot of the zsh completions exported in the zig-out directory:
<img width="1032" alt="Screenshot 2024-12-16 at 7 31 24 PM" src="https://github.com/user-attachments/assets/cc519758-7ec9-4e06-94ae-0431e4d99a1c" />

**Comment by jparise** [comment, marked as answer]
#2988 will fix this.

**Reply by phillip-hirsch** [reply]
Thank you!
